jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/CarouselTests.java

```diff
@@ -0,0 +1,150 @@
+			                          prevText + "\n" + nextText;
+
+	//Carousel slides only tests
+	@Test
+	public void getSlidesTextTest() {
+		carouselWithSlidesOnly.is().text(firstSlideText);
+		carouselWithSlidesOnly.is().text(secondSlideText);
+	}
+	
+	//Carousel with controls tests
+	@Test
+	public void prevTest() {
+		carouselWithControls.prev();	
+		carouselWithControls.is().text(secondSlideFullText);
+		
+		carouselWithControls.prev();	
+		carouselWithControls.is().text(firstSlideFullText);
+		
+		carouselWithControls.prevControl().is().text(prevText);
+	}
+	
+	@Test
+	public void nextTest() {
+		carouselWithControls.next();	
+		carouselWithControls.is().text(firstSlideFullText);
+		
+		carouselWithControls.next();	
+		carouselWithControls.is().text(secondSlideFullText);
+		
+		carouselWithControls.nextControl().is().text(nextText);
+	}
+	
+	//Carousel with indicators tests
+	@Test
+	public void selectTest() {
+		carouselWithIndicators.select(1);	
+		carouselWithIndicators.is().text(firstSlideFullText);
+		carouselWithIndicators.indicators().get(0).is().selected();
+		carouselWithIndicators.indicators().get(1).is().deselected();
+		
+		carouselWithIndicators.select(3);	
+		carouselWithIndicators.is().text(thirdSlideFullText);
+		carouselWithIndicators.indicators().get(2).is().selected();
+		carouselWithIndicators.indicators().get(0).is().deselected();
+		
+		carouselWithIndicators.select(2);	
+		carouselWithIndicators.is().text(secondSlideFullText);
+		carouselWithIndicators.indicators().get(1).is().selected();
+		carouselWithIndicators.indicators().get(2).is().deselected();
+    }
+	
+	//Carousel with captions tests
+	@Test
+	public void captionTest() {
+		carouselWithCaptions.select(1);
+		carouselWithCaptions.is().text(firstSlideWithLabelText);
+		
+		carouselWithCaptions.select(2);
+		carouselWithCaptions.is().text(secondSlideWithLabelText);
+
+		carouselWithCaptions.select(3);
+		carouselWithCaptions.is().text(thirdSlideWithLabelText);
+	}
+	
+	// Carousel fade tests
+	@Test
+	public void fadePrevTest() {
+		carouselWithFadeTransition.prev();	
+		carouselWithFadeTransition.is().text(thirdSlideFullText);
+		
+		carouselWithFadeTransition.prev();	
+		carouselWithFadeTransition.is().text(secondSlideFullText);
+		
+		carouselWithFadeTransition.prevControl().is().text(prevText);
+	}
+	
+	@Test
+	public void fadeNextTest() {
+		carouselWithFadeTransition.next();	
+		carouselWithFadeTransition.is().text(secondSlideFullText);
+		
+		carouselWithFadeTransition.next();	
+		carouselWithFadeTransition.is().text(thirdSlideFullText);
+		
+		carouselWithFadeTransition.nextControl().is().text(nextText);
+	}
+	
+	// Carousel with interval tests
+	@Test
+	public void intervalTest() {
+		WebPage.reload();
+		durationMoreThan(5, 
+				() -> carouselWithCustomInterval.is().text(secondSlideFullText));
+		durationMoreThan(carouselWithCustomInterval.interval()/1000, 
```

Comment:
looks like we need a local variable for carouselWithCustomInterval.interval()/1000 and for 5